### DISCUSSION TITLE
How to wire in CHANGELOG generation

### DISCUSSION THREAD

**Discussion by noherczeg** [ghiscoding/lerna-lite#327] [Q&A] [answered]
Hi,

First of all, thank you for the lib!

My question is: is it possible to add a small guide to the docs which explains how can we generate changelogs for each `version`? I know some keywords, like "conventional commits" etc, but back in the old days there were some challenges with Lerna, and I'm not sure how can we achieve the same.

Based on the repo package, scripts, etc it's also not trivial for me how you did it.

Thanks,
Norbert

**Comment by ghiscoding** [comment, marked as answer]
You need to enable conventional changelog, I don't think it works without the option enabled. I would suggest you watch this Youtube video: https://www.youtube.com/watch?v=p6qoJ4apCjA
